tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/PostResourceIntTest.java

```diff
@@ -397,6 +399,164 @@ public void shouldAllowMMultipleOtherSpecialties() throws Exception {
         .isEqualTo(PostSpecialtyType.OTHER);
   }
 
+  @Test
+  @Transactional
+  public void shouldFailToUpdatePostIfSubspecialtyIsNotASubspecialty() throws Exception {
```

Comment:
Again, possibly just a point of clarity, but it might be worth trying to distinguish from the postSpecialtyType and the  specialtyType to make it a bit easier to understand the test, e.g. calling this 'shouldFailToUpdatePostIfSubspecialtyIsNotOfTypeSubspecialty'

Reply:
I've renamend all 4 tests to look like this.